### PR TITLE
chore: Reformat with HTML Tidy

### PR DESCRIPTION
**Pull request by nondebug**
Updated tidyconfig.txt to specify indent-spaces and ran HTML tidy:

  tidy -config tidyconfig.txt -m index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,318 +1,325 @@
...
+    <section data-dfn-for="Navigator">
+      <h2>
+        Extensions to the {{Navigator}} interface
+      </h2>
+      <pre class="idl">
     [Exposed=Window, SecureContext]
     partial interface Navigator {
       [SameObject] readonly attribute Serial serial;
     };
   </pre>
```
Comment (reillyeon) [inline index.html:36]: Can to fix `<pre>` blocks so that the opening and closing tags are indented to the same level?

File: index.html [hunk 2]
```diff
@@ -1,318 +1,325 @@
...
-  instance of the {{Serial}} object.
-</section>
-
-<section data-dfn-for="WorkerNavigator">
-  <h2>Extensions to the {{WorkerNavigator}} interface</h2>
-
-  <pre class="idl">
+      <h3>
+        <dfn>serial</dfn> attribute
+      </h3>When getting, the {{Navigator/serial}} attribute always returns the
```
Comment (reillyeon) [inline index.html:39]: We probably need to add `<p>` tags around text like this so that the tool doesn't collapse it onto the previous line.

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (nondebug) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (nondebug) [reply index.html:39]: I've added `<p>` tags where they seem appropriate

**Review by reillyeon — APPROVED** [review]

**Pull request merged**